src/cascadia/TerminalSettingsEditor/EnumEntry.idl

```diff
@@ -3,9 +3,10 @@
 
 namespace Microsoft.Terminal.Settings.Editor
 {
-    [default_interface] runtimeclass EnumEntry : Windows.UI.Xaml.Data.INotifyPropertyChanged
+    [default_interface] runtimeclass EnumEntry : Windows.UI.Xaml.Data.INotifyPropertyChanged, Windows.Foundation.IStringable
     {
         String EnumName { get; };
         IInspectable EnumValue { get; };
+        String ToString();
```

Comment:
Do we actually need to define this method here? I thought in midl if you just had a class that implements an interface, it implicitly defines the implemented methods. 

Case in point: `TerminalSettings` and `IControlSetting`&`ICoreSettings`. `TerminalSettings` doesn't re-declare that "Yes, I implement everything in the interfaces I said I am"

Reply:
Good catch, I totally forgot about that tbh. Removing.